sched/scheduler/stateful_scheduler.go

```diff
@@ -44,19 +44,22 @@ const DefaultDefaultTaskTimeout = 30 * time.Minute
 const DefaultTaskTimeoutOverhead = 15 * time.Second
 
 // Number of different requestors that can run jobs at any given time.
-const DefaultMaxRequestors = 100
+const DefaultMaxRequestors = 10
```

Comment:
Reasoning behind this?

Reply:
This lines up better with CloudExec size during dev testing. I'll add a comment indicating that these are example numbers for a relatively small cluster.